org.eclipse.lsp4e.debug/src/org/eclipse/lsp4e/debug/launcher/JsonParserWithStringSubstitution.java

```diff
@@ -0,0 +1,87 @@
+package org.eclipse.lsp4e.debug.launcher;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.variables.IStringVariableManager;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+
+public class JsonParserWithStringSubstitution {
```

Comment:
This makes it public API, which is probably undesired. Can you please try to move it to an internal package?

Reply:
Done